api/src/main/java/org/openmrs/ui/framework/fragment/FragmentFactory.java

```diff
@@ -76,11 +84,7 @@
     List<PossibleFragmentActionArgumentProvider> possibleFragmentActionArgumentProviders;
 
 	private boolean developmentMode = false;
-	

```

Comment:
I think because 'conventionally' most java developers like to put static variables  first